docs/upgrade.html

```diff
@@ -19,6 +19,22 @@
 
 <script id="upgrade-template" type="text/x-handlebars-template">
 
+<h4><a id="upgrade_3_5_1" href="#upgrade_3_5_1">Upgrading to 3.5.1 from any version 0.8.x through 3.4.x</a></h4>
+    All upgrade steps remain same as <a id="upgrade_3_5_0" href="#upgrade_3_5_0">upgrading to 3.5.0</a>
+    <h5><a id="upgrade_351_notable" href="#upgrade_351_notable">Notable changes in 3.5.1</a></h5>
+    <ul>
+        <li>
+            Upgrade the dependency, snappy-java, to a version which does is not vulnerable to
```

Comment:
Should we use the past tense `Upgraded`/`Fixed` instead? Otherwise it kind of seems like _users_ need to upgrade the snappy-java dependency.

Reply:
done.